azure-pipelines.yml

```diff
@@ -2,72 +2,101 @@ trigger:
   branches:
     include:
     - refs/tags/v*
+    
+jobs:
+- job: Build
+  pool:
+    name: Azure-IoT-EdgeExperience-1ES-Hosted-Linux
+    demands:
+      - ImageOverride -equals MMSUbuntu18.04TLS
 
-pool:
-  name: Azure-IoT-EdgeExperience-1ES-Hosted-Linux
-  demands:
-    - ImageOverride -equals MMSUbuntu18.04TLS
+  steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '14.x'
+        checkLatest: true
 
-steps:
-  - task: NodeTool@0
-    inputs:
-      versionSpec: '14.x'
-      checkLatest: true
+    - script: |
+          npm install yo
+          npm install 
+      displayName: 'npm install'
+      
+    - script: |
+          npm run lint 
+      displayName: 'npm lint'
 
-  - script: |
-        npm install yo
-        npm install 
-    displayName: 'npm install'
-    
-  - script: |
-        npm run lint 
-    displayName: 'npm lint'
+    - task: Npm@1
+      displayName: "npm Packaging"
+      inputs:
+          command: 'custom'
+          customCommand: 'pack'
+
+    - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
+      displayName: 'SBOM Generation Task'
+      inputs:
+        BuildDropPath: '$(Build.ArtifactStagingDirectory)'
```

Comment:
>$(Build.ArtifactStagingDirectory) [](http://example.com/codeflow?start=24&length=33)

This task needs to run on $(Build.ArtifactStagingDirectory)/Package, where the tgz file is located